modules/installation/pages/kubernetes-vm.adoc

```diff
@@ -0,0 +1,40 @@
+= Kubernetes VM
+:toc:
+:toclevels: 3
+
+WARNING: This page is a work in progress. *This page will be automated, instead the user will be prompted before rebooting
+to disk or right after.*
```

Comment:
The second sentence in this warning is confusing, was it intentional or copy-pasted?